spec/rubocop/cop/performance/zip_for_array_wrapping_spec.rb

```diff
@@ -0,0 +1,389 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::ZipForArrayWrapping, :config do
+  context 'when using map with array literal' do
+    it 'registers an offense and corrects to use zip with no arguments' do
+      expect_offense(<<~RUBY)
+        [1, 2, 3].map { |id| [id] }
+                  ^^^^^^^^^^^^^^^^^ Use `zip` instead of `map { |id| [id] }`.
```

Comment:
Can you update the message in this test?

```suggestion
                  ^^^^^^^^^^^^^^^^^ Use `zip` instead.
```